docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -129,13 +129,13 @@
                 node_filesystem_readonly{%(nodeExporterSelector)s,%(fsSelector)s,%(fsMountpointSelector)s} == 0
               )
             ||| % $._config,
-            'for': '1h',
+            'for': '15m',
```

Comment:
See above. Curious why we had this different from the file space.

Reply:
AFAIK, the idea of for is to be sure that problem is permanent and won't resolve itself. At the same time, if it is a real actionable problem, 1h could be too long period to wait. Plus, if disk is rather small and filling up fast, it may fill up to 100% faster than 1h.